roles/openshift-applier/tasks/prep-access.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: "Create temporary dir"
+  tempfile:
+    state: directory
+    prefix: "openshift-{{ ansible_host }}"
+  register: openshift_config_tempdir
+  notify: 
+    - "Cleanup OpenShift config"
+
+- name: "Define temporary config dir"
+  set_fact:
+    kubeconfig: "{{ openshift_config_tempdir.path }}/config"
+
+- name: "Capture OpenShift Config based on token"
+  command: >
+    oc login --token={{ openshift_token }} {{ openshift_insecure | ternary('--insecure-skip-tls-verify', '') }} {{ openshift_login_url }}
```

Comment:
This will cause pre-existing applier inventories to fail if not set. I would suggest two things:

- first, make this optional. i.e. skip it if these variables are not set
- set a default value for `openshift_insecure` so that it is not required.

Reply:
@etsauer yes, there are multiple things that need to be polished before this can be merged. Main intent for the PR was to have a discussion starting point around the concept (and if it's worth spending time on it)